Lesson24-signup-form/assets/assets/ezgi/index.htm

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Document</title>
+  <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+    <div id="container">
+  <div class="form-container">
+    <form id="myForm" novalidate>
+
+<h1>Stay Updated!</h1>
```

Comment:
You don't need to place h1, p, and ul  inside form elment. I'd say it would be better to have structure like this instead:
```
<div>
  <div>
    <h1></h1>
    <p></p>
    <ul>...</ul>
  </div>
  <form>...</form
</div>
```